pkg/telemetry/metrics.go

```diff
@@ -1,80 +1 @@
 package telemetry
-
```

Comment:
Do we need this file at all anymore?

Reply:
Not right now, but I thought I'd leave it so that we can start adding custom metrics in this file. But we can get rid of it in favor of adding them in later 👍

Reply:
Removed for now. We can introduce custom metrics in each package that defines them 👍